Packs/QintelFeed/Integrations/QintelQSentryFeed/test_data/anon_feed.jsonl

```diff
@@ -0,0 +1,5 @@
+{"ip_address": "101.100.146.147", "service_type": "tor", "ip_type": "4", "service_name": "tor_exitnodes", "criminal": 0, "comment": "This IP address has been associated with the TOR network.", "cdn": 1}
+{"ip_address": "101.99.90.171", "service_type": "tor", "ip_type": "4", "service_name": "tor_exitnodes", "criminal": 1, "comment": "This IP address has been associated with the TOR network.", "cdn": 0}
+{"ip_address": "102.130.113.37", "service_type": "tor", "ip_type": "4", "service_name": "tor_exitnodes", "criminal": 0, "comment": "This IP address has been associated with the TOR network.", "cdn": 0}
+{"ip_address": "102.130.113.9", "service_type": "tor", "ip_type": "4", "service_name": "tor_exitnodes", "criminal": 0, "comment": "This IP address has been associated with the TOR network.", "cdn": 0}
+{"ip_address": "103.228.53.155", "service_type": "tor", "ip_type": "4", "service_name": "tor_exitnodes", "criminal": 0, "comment": "This IP address has been associated with the TOR network.", "cdn": 0}
```

Comment:
Please change the file to json instead of jsonl

Reply:
done